[Bug] [datavines-ui] 前端编译打包失败 TS2739: Type '{}' is missing the following properties from type

### Search before asking

- [X] I had searched in the [issues](https://github.com/datavines-ops/datavines/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

使用 mvn clean package -Prelease 命令构建项目到 datavines-ui 模块时报错，详细报错信息在附件，主要是下面这种类型错误
`[INFO] ERROR in ./src/view/Main/Warning/SLAsNoticeList/SLAsList.tsx:147:36
[INFO] TS2739: Type '{}' is missing the following properties from type 'Pick<AntdIconProps, "type" | "id" | "cite" | "data" | "form" | "label" | "slot" | "span" | "style" | "summary" | "title" | "download" | "children" | "size" | "pattern" | ... 348 more ... | "twoToneColor">': onPointerEnterCapture, onPointerLeaveCapture
`

[error_info.txt](https://github.com/datavane/datavines/files/14610372/error_info.txt)



### DataVines Version

dev 分支的 74b7ba4fe67eb8d62e61c64f37e681e8db731cda 提交

### DataVines Config

```conf
默认
```


### Running Command

```shell
mvn clean package -Prelease
```


### Error Exception

```log
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  41.409 s
[INFO] Finished at: 2024-03-15T09:45:45+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.1:npm (npm run build:prod) on project datavines-ui: Failed to run task: 'npm run build:prod' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :datavines-ui
```


### Engine Type

_No response_

### Java Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

--- a/datavines-ui/package.json
+++ b/datavines-ui/package.json
@@ -18,6 +18,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
+    "@ant-design/icons": "^5.3.2",
     "@monaco-editor/loader": "^1.3.2",
     "ahooks": "^3.7.4",
     "antd": "^5.0.5",

